ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh

```diff
@@ -246,8 +246,8 @@ function start_mon {
       exit 1
     fi
 
-    if [ ! -e /etc/ceph/monmap ]; then
-      echo "ERROR- /etc/ceph/monmap must exist.  You can extract it from your current monitor by running 'ceph mon getmap -o /etc/ceph/monmap' or use a KV Store"
+    if [ ! -e /etc/ceph/monmap-${CLUSTER} ]; then
```

Comment:
For existing clusters, this file will never exit, can we keep the previous check and then `mv` with the desired cluster name? Then we can safely move to this check and just do nothing.